skimage/feature/_canny.py

```diff
@@ -69,9 +201,10 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None,
     mask : array, dtype=bool, optional
         Mask to limit the application of Canny to a certain area.
     use_quantiles : bool, optional
-        If True then treat low_threshold and high_threshold as quantiles of the
-        edge magnitude image, rather than absolute edge magnitude values. If True
-        then the thresholds must be in the range [0, 1].
+        If True then treat low_threshold and high_threshold as
```

Comment:
Minor: it is normally better to put backticks around True False and variables, but I never remember if simple or double backticks must be used.